software/MainboardCommunication.py

```diff
@@ -0,0 +1,17 @@
+import struct
+import serial
+import time
+
+disable_failsafe = 0
+speed1 = 20
+speed2 = 0
+speed3 = -20
+thrower_speed = 00
+
+andmed = struct.pack('<hhhHBH', speed1, speed2, speed3, thrower_speed, disable_failsafe, 0xAAAA)
+ser = serial.Serial('/dev/ttyACM0')  # open serial port
```

Comment:
Serial port might change if there are more serial port devices connected or if mainboard is disconnected and reconnected.

`serial.tools.list_ports` can be used to find the correct serial port device path by vendor and product IDs for example.